libs/sdk-ui-kit/src/Header/generateHeaderAccountMenuItems.ts

```diff
@@ -18,15 +18,17 @@ export function generateHeaderAccountMenuItems(
     uiSettings: IUiSettings, // bootstrapResource.settings
     workspaceId?: string, // parsed from bootstrapResource.current.project.links.self
     showOnlyLogoutItem?: boolean,
+    featureFlags?: ISettings,
 ): IHeaderMenuItem[] {
     const { canInitData } = workspacePermissions;
     const { displayAccountPage } = uiSettings;
     const accountMenuItems: IHeaderMenuItem[] = [];
+    const workspaceRef = featureFlags?.enableRenamingProjectToWorkspace ? "workspace" : "project";
 
     const accountItem = {
         key: "gs.header.account",
         className: "s-account",
-        href: `/#s=/gdc/projects/${workspaceId}|accountPage|`,
+        href: `/#s=/gdc/${workspaceRef}s/${workspaceId}|accountPage|`,
```

Comment:
If `workspaceRef` is already in plural form then we don't need to add a `s` at the end here.

Reply:
I have changed it thanks